run_pipeline.sh

```diff
@@ -1,3 +1,3 @@
 #!/usr/bin/env sh
 
-./packages/cli/dist/bin/pipcook run ./test/pipelines/$1.json --verbose
+./packages/cli/dist/bin/pipcook run "./test/pipelines/$1.json" --verbose
```

Comment:
Which case do you make this change?

Reply:
Filename with space.

Reply:
Do we have pipeline filename with spaces?

Reply:
It's depended on the user.